.github/workflows/nidx_binding_release.yml

```diff
@@ -1,73 +1,34 @@
-name: Release NucliaDB Node
+name: Release nidx binding
 
 on:
   pull_request:
     branches:
       - main
     paths:
-      - ".github/workflows/nucliadb_node_release.yml"
-      - "nucliadb_node_binding/**"
-      - "Cargo.lock"
-      - "Cargo.toml"
-      - "nucliadb_core/**"
-      - "nucliadb_node/src/**"
-      - "nucliadb_node/Cargo.toml"
-      - "nucliadb_paragraphs[0-9]*/**"
-      - "nucliadb_relations[0-9]*/**"
-      - "nucliadb_texts[0-9]*/**"
-      - "nucliadb_vectors/**"
+      - "nidx"
+      - "nucliadb_protos"
   push:
     branches:
       - main
     paths:
-      - ".github/workflows/nucliadb_node_release.yml"
-      - "nucliadb_node_binding/**"
-      - "Cargo.lock"
-      - "Cargo.toml"
-      - "nucliadb_core/**"
-      - "nucliadb_node/Cargo.toml"
-      - "nucliadb_node/src/**"
-      - "nucliadb_paragraphs[0-9]*/**"
-      - "nucliadb_relations[0-9]*/**"
-      - "nucliadb_texts[0-9]*/**"
-      - "nucliadb_vectors/**"
+      - "nidx"
+      - "nucliadb_protos"
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
   build:
-    name: Build on ${{ matrix.platform || matrix.os }} (${{ matrix.target }} - ${{ matrix.manylinux || 'auto' }})
+    name: Build on ${{ matrix.os }} (${{ matrix.target }})
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu, macos]
-        target: [x86_64, aarch64]
-        manylinux: [auto]
         include:
           - os: ubuntu
-            platform: linux
-          - os: macos
-            target: aarch64
-          # - os: ubuntu
-          #   platform: linux
-          #   target: i686
-          # GCC 4.8.5 in manylinux2014 container doesn't support c11 atomic
-          # we use manylinux_2_24 container for aarch64 and armv7 targets instead,
-          - os: ubuntu
-            platform: linux
-            target: aarch64
-            container: messense/manylinux_2_24-cross:aarch64
-          # musllinux
-          - os: ubuntu
-            platform: linux
             target: x86_64
-            manylinux: musllinux_1_1
-          - os: ubuntu
-            platform: linux
+          - os: macos
```

Comment:
Removed a lot of platforms that we were not using and that were going to be a pain to make work with newer rust compilers. Left the two main platforms: linux on x86_64 and mac on arm64 and the source distribution so others can install from source.